pandas/io/formats/html.py

```diff
@@ -57,8 +57,11 @@ def __init__(
         self.bold_rows = self.fmt.bold_rows
         self.escape = self.fmt.escape
         self.show_dimensions = self.fmt.show_dimensions
-        if border is None:
+        if border is None or border is True:
```

Comment:
so if border is not passed then we get a border by default?

Reply:
Correct - this was the prior behavior and it was my intent to preserve that as much as possible. The extension to allow True is to ensure that boolean flags passed to `border` flip properly. Sp `border=is_hidden` will result in the default behavior if `is_hidden is True`.

Reply:
Since this is deprecated HTML it may, at no notice, stop rendering in a browser, or may not do so now in certain browsers. If making changes here I would recommend removing border by default. 

When (if) DataFrame.to_html uses the Styler implementation this border property will either not be available, or will automatically implement CSS solutions which are the more modern way to execute this.

Reply:
@attack68 well that's a future action, this is valid as of now.